iCubGenova04/icub_wbd.xml

```diff
@@ -47,7 +47,7 @@
     <xi:include href="wrappers/inertials/head-inertials_wrapper.xml" />
     <xi:include href="wrappers/inertials/head-inertials_wrapper-deprecated.xml" />
     <xi:include href="hardware/inertials/head-inertial.xml" />
-    <xi:include href="hardware/inertials/waist-inertial.xml" /> 
+    <!--xi:include href="hardware/inertials/waist-inertial.xml" /--> 
```

Comment:
This change is fishy. @prashanthr05 any opinion?

Reply:
Sometimes it required to run the waist-inertial independently from the `yarprobotinterface` to ensure that the device was not stopping abruptly (See https://github.com/robotology/yarp-device-xsensmt/issues/8). Which is why this change exists. We should revert back to the default where this line remains uncommented.

Reply:
Looks like I had forgotten to uncomment and leave it at default. I apologize. I suppose I was the only one in the meantime using the XSens IMU, so it shouldn't have affected any other experiments.